hmy/api_backend.go

```diff
@@ -307,6 +308,41 @@ func (b *APIBackend) GetValidatorInformation(addr common.Address) *staking.Valid
 	return &val.Validator
 }
 
+var (
+	two = big.NewInt(2)
+)
+
+// GetMedianRawStakeSnapshot  ..
+func (b *APIBackend) GetMedianRawStakeSnapshot() *big.Int {
+	candidates := b.hmy.BlockChain().ValidatorCandidates()
```

Comment:
This only works on beacon chain. So better just expose this rpc for beacon chain explorer.